iroh/Cargo.toml

```diff
@@ -6,5 +6,22 @@ license = "Apache-2.0/MIT"
 readme = "README.md"
 description = "coming soon"
 
+[features]
+testing = []
+
 [dependencies]
+cid = "0.8.5"
+iroh-resolver = { path = "../iroh-resolver" }
+iroh-rpc-types = { path = "../iroh-rpc-types" }
+iroh-rpc-client = { path = "../iroh-rpc-client" }
+anyhow = "1"
+async-trait = "0.1.53"
+tokio = { version = "1" }
+bytes = "1.1.0"
+libp2p = "0.49"
+tracing = "0.1.34"
+futures = "0.3.21"
+async-stream = "0.3.3"
+mockall = "0.11.2"
```

Comment:
shouldn't this be optional and only enabled when the feature `testing` is enabled?

Reply:
Good point. I learned another cargo trick with optional packages, hopefully it looks good now.